### PR TITLE
Beginner's Guide

### PR DESCRIPTION
**Pull request by mckenzietsean**
### Summary 
Added Beginner's Guide for users to use
### Details
- Feel free to change the text at any point
- Text is located in guideFunc.tsx
### How to test?
1. Log in and open the editor
2. Click on the ? icon in the bottom of the side bar
3. Go through guide
4. ???
5. Profit
### Checks 
- [ ] Tested Changes
- [ ] Stakeholder Approval

### CODE REVIEW

File: src/components/Editor/Guide.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            open={showGuide}
+            onClose={toggleGuide}
+            closeAfterTransition
+            BackdropComponent={Backdrop}
+            BackdropProps={{
+                timeout: 500,
+            }}
+            >
+            <Fade in={showGuide}>
+                <Box sx={style}>
```
Comment (shushengfuqin) [inline src/components/Editor/Guide.tsx:50]: the sx is not passing, 
It's a problem with typescript. By having style directly typed in side will be fine.
i.g. sx ={{position: 'absolute',
  top: '50%',
  left: '80%',
  transform: 'translate(-50%, -50%)',
  width: 400,
  bgcolor: 'background.paper',
  background: '#3f3d56',
  borderRadius: '15px',
  boxShadow: 24,
  p: 4, 
  fontFamily: 'Trebuchet MS'
  }}

**Review by shushengfuqin — CHANGES_REQUESTED** [review]
Just change that part, or if you found another way to solve it. pass the ci/cd then it's good for me. The functions looks perfect

File: src/components/Editor/Guide.tsx (hunk 1 above)
Comment (mckenzietsean) [reply src/components/Editor/Guide.tsx]: Kinda odd that typescript can't handle this but it should be fixed now.

**Review by shushengfuqin — APPROVED** [review]
lgtm

**Pull request merged**